### PR TITLE
Temp Fix for Delayed Buffering of Stop Response during Device Construction on Raspberry Pi

### PR DESCRIPTION
**Pull request by dcyoung**
This a temporary solution to the problems described in #73. See the issue for details.

It would be nice to know the root cause of the bug, but this temporary fix will allow development of the 3d-scanner to commence.

### CODE REVIEW

File: libsweep/src/sweep.cc [hunk 1]
```diff
@@ -218,11 +218,25 @@ void sweep_device_stop_scanning(sweep_device_s device, sweep_error_s* error) {
     return;
   }
 
-  // Wait some time, for the device to register the stop cmd and stop sending data blocks
+  // Wait some time for a few reasons:
+  // 1. allow time for the device to register the stop cmd and stop sending data blocks
+  // 2. allow time for slower performing devices (RaspberryPi) to buffer the stop response
+  // 3. allow a grace period for the device to perform its logging routine before sending a second stop command
```
Comment (daniel-j-h) [inline libsweep/src/sweep.cc]: As said in the ticket, if we can just make the timeout below a bit longer I would go for that. Even hundreds of milliseconds are fine for construction if it makes things easier for us.

Comment (dcyoung) [inline libsweep/src/sweep.cc]: Check [ticket](https://github.com/scanse/sweep-sdk/issues/73) for response.

**Comment by dcyoung** [comment]
Seems that this is definitely an issue other Raspberry Pi's as well, and that this fix is resolving the issue. See #88 

@daniel-j-h Perhaps there is a better fix applicable to `unix/serial.cc`. I didn't write that code, but if you have any ideas for a possible source or fix let me know and I'd be happy to try and debug the `unix/serial.cc`.

In the meantime though, I'd like to get this merged as I think the Pi might be a common use case.  Is there any reason we shouldn't merge this now, and bring a better fix in a later PR?

**Comment by daniel-j-h** [comment]
Urgh - great that it's fixed upstream. Feel free to merge if it's needed until Raspbian picks the fix up.

But please ticket this for later removal and maybe link it even from the source code. I don't want to add too many hacks for platform quirks.

There _never_ should be a downside of doing this, right? Otherwise we need to detect the Raspberry env. with CMake and only conditionally enable the workaround.

**Comment by daniel-j-h** [comment]
@dcyoung still want to bring this in? I just added some release docs and a changelog [here](https://github.com/scanse/sweep-sdk/pull/97). Feel free to merge the release docs pull request first and then make a release with these Raspberry specific changes.

**Comment by dcyoung** [comment]
@daniel-j-h  sounds good. I was dragging my feet a little, waiting to get a better estimate on the release timeframe for the Raspbian build.... but I think its silly to wait on it. The rPi will be a relatively common use case and I'd prefer to avoid confusion for those who aren't aware of this branch. 

As for the release docs, I made some comments there. Let me know what you think, and if you want to keep things simple I'll merge them in that order. Additionally, should we merge #98 (API cleanup PR) later, or before all these others? I'm wondering if the code shuffling will make rebasing all the other PRs more cumbersome.

**Comment by daniel-j-h** [comment]
I think order does not really matter as there probably will be no conflicts.

**Pull request merged**